.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,6 @@
+> **NOTE:** Kedro datasets are moving from `kedro.extras.datasets` to `kedro-datasets` package in
```

Comment:
```suggestion
> **NOTE:** Kedro datasets are moving from `kedro.extras.datasets` to a separate `kedro-datasets` package in
```

Reply:
Added in 66a7dbba2f1cf798037869ebca0552a439f9eb03! Thank you!